api/src/api/opportunities_v1/opportunity_schemas.py

```diff
@@ -308,6 +308,60 @@ class OpportunityV1Schema(Schema):
     updated_at = fields.DateTime(dump_only=True)
 
 
+class OpportunityAttachmentV1Schema(Schema):
+    file_location = fields.String(
+        metadata={
+            "description": "The URL to download the attachment",
+            "example": "https://...",
+        }
+    )
+    mime_type = fields.String(
+        metadata={"description": "The MIME type of the attachment", "example": "application/pdf"}
+    )
+    file_name = fields.String(
+        metadata={"description": "The name of the attachment file", "example": "my_NOFO.pdf"}
+    )
+    file_description = fields.String(
+        metadata={
+            "description": "A description of the attachment",
+            "example": "The full announcement NOFO",
+        }
+    )
+    file_size_bytes = fields.Integer(
+        metadata={"description": "The size of the attachment in bytes", "example": 10012}
+    )
+    opportunity_attachment_type = fields.String(
+        metadata={
+            "description": "The type of attachment",
+            "example": "notice_of_funding_opportunity",
+        }
+    )
+    created_by = fields.String(
+        metadata={
+            "description": "The user who created the attachment",
+        }
+    )
+    updated_by = fields.String(
+        metadata={
+            "description": "The user who last updated the attachment",
+        }
+    )
+
+    legacy_folder_id = fields.Integer(
+        metadata={
+            "description": "The legacy folder ID",
+        }
+    )
```

Comment:
We don't want these fields in the API schema itself - they're just for some internal purposes at the moment for the legacy system.

We do however want the created_at/updated_at timestamp fields for these.

Reply:
Got it, made this change.